editors/emacs/Portfile

```diff
@@ -94,18 +94,19 @@ if {$subport eq $name || $subport eq "emacs-app"} {
 
 if {$subport eq "emacs-devel" || $subport eq "emacs-app-devel"} {
     epoch           1
-    version         20200417
+    version         20200524
 
     fetch.type      git
-    git.url         http://git.savannah.gnu.org/r/emacs.git
-    git.branch      c36c5a3dedbb2e0349be1b6c3b7567ea7b594f1c
+    git.url         https://github.com/emacs-mirror/emacs.git
+    git.branch      e7a3ed8a6dddb6e16c83d27a04dfa6ec8160e580
 
     pre-configure {
         system -W ${worksrcpath} "sh ./autogen.sh"
     }
 
     configure.args-append  --with-json
-    depends_lib-append     port:jansson
+    depends_lib-append     port:jansson \
+                           port:gmp
```

Comment:
Please sort the dependencies alphabetically.

Reply:
None of the the dependencies below are sorted alphabetically, many ports don't do this. This is not documented in Trac either. Did you just make up a new rule?

Reply:
I personally consider this a nicety but not a firm requirement. It does make it easier to scan read a long list of deps, to check for example if one is there, if they are sorted. But for short lists its less an issue. I’m going to accept this, but please just consider this in the future.